runtimes/pythonrt/build.go

```diff
@@ -74,7 +74,13 @@ func (py *pySvc) Build(ctx context.Context, fsys fs.FS, path string, values []sd
 	var art sdktypes.BuildArtifact
 	art = art.WithCompiledData(compiledData)
 
-	exports, err := findExports(py.log, fsys)
+	hash, err := getRequirementsHash(data)
```

Comment:
I'm not sure passing `hash` around is the right way. Maybe add a `pythonExe` field to `py` and populate it?

Reply:
now passing the actual requirements